include/command_macros_base.h

```diff
@@ -6,16 +6,22 @@
  * Each macro packs bytes (B), halfowrds (H) and words (W, for consistency) into a single word
  */
 
-#define CMD_BBBB(a, b, c, d) (_SHIFTL(a, 24, 8) | _SHIFTL(b, 16, 8) | _SHIFTL(c, 8, 8) | _SHIFTL(d, 0, 8))
+#define CMD_BBBB(a, b, c, d) {(_SHIFTL(a, 24, 8) | _SHIFTL(b, 16, 8) | _SHIFTL(c, 8, 8) | _SHIFTL(d, 0, 8))}
 
-#define CMD_BBH(a, b, c) (_SHIFTL(a, 24, 8) | _SHIFTL(b, 16, 8) | _SHIFTL(c, 0, 16))
+#define CMD_BBH(a, b, c) {(_SHIFTL(a, 24, 8) | _SHIFTL(b, 16, 8) | _SHIFTL(c, 0, 16))}
 
-#define CMD_HBB(a, b, c) (_SHIFTL(a, 16, 16) | _SHIFTL(b, 8, 8) | _SHIFTL(c, 0, 8))
+#define CMD_HBB(a, b, c) {(_SHIFTL(a, 16, 16) | _SHIFTL(b, 8, 8) | _SHIFTL(c, 0, 8))}
 
-#define CMD_HH(a, b) (_SHIFTL(a, 16, 16) | _SHIFTL(b, 0, 16))
+#define CMD_HH(a, b) {(_SHIFTL(a, 16, 16) | _SHIFTL(b, 0, 16))}
 
-#define CMD_W(a) (a)
+#define CMD_W(a) {(a)}
```

Comment:
changing these to use a struct initialization syntax could mean they would now belong more in a cutscene-specific header than a general-purpose header? especially with `CMD_F` using a named field (`CutsceneData#f`)

Reply:
Well, yeah. `CMD_F` should be used only with `CutsceneData`. This will allow to _not_ use the asm processor in any non-IDO build in the future, like modding contexts, building with gcc, etc.

Reply:
I agree with Dragorn, this definitely doesn't belong in this header because it's intended as general purpose macros to build commands upon, not just for cutscenes (in fact they are already used in scene commands as well).

For this reason, I think we should leave these macros as is and have wrappers with the braces and all in the cutscene command headers.